examples/js/loaders/OBJLoader2.js

```diff
@@ -0,0 +1,996 @@
+/**
+  * @author Kai Salmen / https://kaisalmen.de
+  * Development repository: https://github.com/kaisalmen/WWOBJLoader
+  */
+
+'use strict';
+
+if ( THREE.OBJLoader2 === undefined ) { THREE.OBJLoader2 = {} }
+THREE.OBJLoader2.version = '1.0.1';
+
+/**
+ * Use this class to load OBJ data from files or to parse OBJ data from arraybuffer or text
+ * @class
+ *
+ * @param {THREE.DefaultLoadingManager} [manager] Extension of {@link THREE.DefaultLoadingManager}
+ */
+THREE.OBJLoader2 = (function () {
+
+	function OBJLoader2( manager ) {
+		this.manager = ( manager == null ) ? THREE.DefaultLoadingManager : manager;
+
+		this.path = '';
+		this.fileLoader = new THREE.FileLoader( this.manager );
+
+		this.meshCreator = new THREE.OBJLoader2.MeshCreator();
+		this.parser = new THREE.OBJLoader2.Parser( this.meshCreator );
```

Comment:
I have fixed the CORS issue. `WWOBJLoader2` and `WWOBJLoader2Director` can now pass crossOrigin to `MTLLoader`.
I read [#10465](https://github.com/mrdoob/three.js/pull/10465) and also set "anonymous" in the examples and now it works fine with rawgit!

Now, I think all issues you brought up are addressed! 👍